core/trino-main/src/main/java/io/trino/execution/DynamicFilterConfig.java

```diff
@@ -41,7 +41,7 @@
     private boolean enableCoordinatorDynamicFiltersDistribution = true;
     private boolean enableLargeDynamicFilters;
 
-    private int smallMaxDistinctValuesPerDriver = 1_000;
+    private int smallMaxDistinctValuesPerDriver = 10_000;
```

Comment:
Could you add benchmark results for streaming mode at 1k scale factor ?
In general, rather than increasing config by 10X because it didn't significantly regress some TPC workloads, I would prefer that we increase this only as much as is necessary for good TPC results. I already 5X'ed this number very recently.
fyi @sopel39

Reply:
This only increases the limits for `BROADCAST` joins where build side is usually small, do you think it may still be significant? 

> I would prefer that we increase this only as much as is necessary for good TPC results

There are 5000+ partitions in partitioned tables in a TPC/DS sf10000 schema. There are also plenty of joins in TPC-DS based on date dimension. This limit increase ensures the filters are always available.